Pod/Classes/public/error/Error.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  Error.swift
+//  Pods
+//
+//  Created by Jake Young on 6/14/16.
+//
+
+import Foundation
+
+/// Debugging levels for WistiaKit
+public enum DebuggingLevel {
+    
+    /// Shhh, fust be silent.
+    case None
+    
+    /// Print the high level activity and errors.
+    case Friendly
+    
+    /// Print all the things.
+    case Annoying
+}
+
+internal func printErrorForDebuggingLevel(debuggingLevel debuggingLevel: DebuggingLevel, message: String? = nil, error: NSError? = nil) {
```

Comment:
I'd like to follow the Swift 3 [guidelines for APIs](https://swift.org/documentation/api-design-guidelines/)

In this case, let's drop `DebuggingLevel` from the function name as it will already be included as the name of the first parameter.